configure.ac

```diff
@@ -40,7 +40,7 @@ AS_IF([test "$host" != "$build"], [
   ( CPPFLAGS="$HOST_CPPFLAGS"; \
     CFLAGS="$HOST_CFLAGS"; \
     LDFLAGS="$HOST_LDFLAGS"; \
-    cd hosttools; "${abssrc}/configure" "--host=$build" "--build=$build"; )
+    cd hosttools; "${abssrc}/configure" "--host=$build" "--build=$build" "--enable-clippy-only" "--disable-nhrpd" "--disable-vtysh"; )
```

Comment:
Are `--disable-nhrpd` and `--disable-vtysh` actually required on a non-host build?  This seems like a bit of your specialized build has leaked in here?

Reply:
I don't want to have unnecessary dev libs installed on the host, almost anything should be used from the build environment and the libs needed by nhrpd and vtysh aren't installed on host by default

Reply:
@lucize In your build you do not want them, but other people might.  If you are cross-compiling I would expect it is surprising that nhrpd and vtysh stopped being compiled, right?

Reply:
no, it doesn't work that way, first run, the host tool clippy will be compiled, so I don't need anything more to be installed on the host build system, on the second stage everything will be compiled as per user option, so if someone wants nhrpd they will have libcares in the dependency and the build system will build a target libcares that doesn't need to be installed on the build host.
am I clear enough ?